opentracing-mock/src/test/java/io/opentracing/mock/MockTracerTest.java

```diff
@@ -156,16 +157,21 @@ public void testTextMapPropagatorTextMap() {
 
             SpanContext extract = tracer.extract(Format.Builtin.TEXT_MAP, new TextMapExtractAdapter(injectMap));
 
-            tracer.buildSpan("bar")
+            Span childSpan = tracer.buildSpan("bar")
                     .asChildOf(extract)
-                    .start()
-                    .finish();
+                    .start();
+            childSpan.setBaggageItem("barbag", "baritem");
+            childSpan.finish();
         }
         List<MockSpan> finishedSpans = tracer.finishedSpans();
 
         Assert.assertEquals(2, finishedSpans.size());
         Assert.assertEquals(finishedSpans.get(0).context().traceId(), finishedSpans.get(1).context().traceId());
         Assert.assertEquals(finishedSpans.get(0).context().spanId(), finishedSpans.get(1).parentId());
+        Assert.assertEquals(finishedSpans.get(0).getBaggageItem("foobag"), "fooitem");
```

Comment:
Could you please change the order of these arguments to align with`Assert.assertEquals(expected, actual)`. (lines 173, 174 too)